paas/install_client.go

```diff
@@ -211,13 +211,24 @@ func (c *InstallClient) handleExtensions(action string, extensions []string, opt
 			if err != nil {
 				return errors.New(fmt.Sprintf("Failed to install extension %s: %s", extension.Name, err.Error()))
 			}
+
+			c.ui.Note().Msg(fmt.Sprintf("Registering extension '%s'...", extension.Name))
+			err = extension.Register(c.kubeClient, c.ui, options)
```

Comment:
Not all installed extensions need to be registered with FuseML (e.g. cert-manager, KNative).

Reply:
Hm... why not?

Of course they are not needed for fuseml-core operations (they won't be needed from workflows for example), but they are useful to know for the installer, so the user can list what extensions are actually installed and potentially delete some.

After all, we are not calling them ML extensions.

Reply:
Hm... why not?

Of course they are not needed for fuseml-core operations (they won't be needed from workflows for example), but they are useful to know for the installer, so the user can list what extensions are actually installed and potentially delete some.

After all, we are not calling them ML extensions.

Reply:
Right now, the only reason you need to register extensions with FuseML is so that you can consume them in workflows. I would say cert-manager has no meaning for ML workflows, and there will be other cases of similar services that are not useful for MLOps. 

Ok, I agree that KNative might be useful for ML workflows, because you could have a workflow component that deploys prediction services directly on top of KNative, so you could advertise/register KNative as a prediction service platform, but that doesn't mean all k8s applications are relevant for ML.

To put it another way, the main purpose of the extension registry is not to provide persistent storage for the installer.

If the fuseml core were also responsible for _installing_ the extensions instead of the installer, like we discussed a few days ago, that would be a completely different story: you would need to store everything in the core.

But if you asked me how can the installer decide which extensions it needs to register with FuseML, I wouldn't be able to give a good answer :)